lib/tasks/install_secrets.rake

```diff
@@ -41,15 +41,16 @@ task :install_secrets, [] do
     response.parameters.each do |parameter|
       # break out the flattened keys and ignore the env name and namespace
       keys = parameter.name.split('/').reject(&:blank?)[num_ignored_key_levels..-1]
-      deep_populate(secrets, keys, parameter.value)
+      value = parameter.type == "StringList" ? parameter.value.split(",") : parameter.value
+      deep_populate(secrets, keys, cast_to_number_if_number(value))
     end
   end
 
   scout_secrets = secrets.delete('scout')
   write_yaml_file("config/scout_apm.yml", {
     production: {
       key: scout_secrets[:license_key],
-      name: "highlights (#{env_name})",
+      name: "quasar-api (#{env_name})",
```

Comment:
do we want this to match the github repro name of event-capture-api?

Reply:
I've gone back and forth on that, to be honest. I see value in both - not too familiar w/ scout, who is reading that data, and what will mislead them the least?

Reply:
There's all of "quasar-api" "events-api" "event-capture-api" in various places in the code. I'm trying to stick w/ Quasar for thing that label more than one part of the system. i.e. both the kafka cluster (MSK) and the api server are part of quasar.